### ISSUE TITLE
Desktop icons change place when new icon is added

### ISSUE THREAD

**Issue by brzegorz** [lxqt/lxqt#1635]
Actual behaviour: 
    When I place new shortcut on the desktop, some others change their position, or the new one isn't placed where I've dragged it, but somewhere else. It happens especially when swapping two icons.

Desired behaviour: 
    None of the icons change places unless it's explicitly dragged somewhere else. This includes swapping it with other icon

**Comment by tsujan** [comment]
> None of the icons change places unless it's explicitly dragged somewhere else

When you place an item *where other items exist*, those items should make room for it. That's the correct behavior.

> ... or the new one isn't placed where I've dragged it, but somewhere else.

Already fixed in the latest git.

**Comment by tsujan** [comment]
And if you mean when you just paste an item and it goes before other items, check right click -> Sorting.

**Comment by brzegorz** [comment]
Yeah, I haven't been precise enough about placing item where other item exist.

Everything is fine as I'm concerned, thank you for swift response. I'd just like to point out that the behaviour with icons being sorted by default, with new icons being sorted, and explicitly moved icons being locked in place is really confusing at first glance, especially on Lubuntu. I've achieved desired state by manually dragging all of the icons somewhere. Sorting itself is neat - I haven't used desktop shortcuts too much in a long time, I've opened this issue to help a friend.

Thanks again!
